.github/actions/setup-build/action.yml

```diff
@@ -15,16 +15,19 @@ runs:
 
   steps:
   - name: Set up Python
+    if: ${{ runner.arch == 'X64' }}
     uses: actions/setup-python@v4
     with:
       python-version: '3.11'
 
   - name: Install MLIR Python depends
+    if: ${{ runner.os != 'Linux' }}
```

Comment:
I'm not sure how this works.  Don't we need to install the Python dependencies to be able to build MLIR's Python bindings?

Reply:
so on macOS we need the system install since we are not dockerized. On Linux both for CI and Release we are dockerized so all the deps are in the docker image. This is a no-op on Linux but it fails to install on arm64 since github doesn't have a Python release for arm64 yet.

Reply:
Ah, I see.  Thanks!